backend/search/base/transactions.go

```diff
@@ -0,0 +1,131 @@
+package base
+
+import (
+	"bytes"
+	"encoding/json"
```

Comment:
just for consistency, let's keep using go_json
```suggestion
		go_json "github.com/goccy/go-json"
```